README.md

```diff
@@ -137,6 +137,40 @@ fastify.register(AutoLoad, {
 })
 ```
 
+fastify-autoload loads folders with route definitions automatically, without explicitly registering them. The folder name is used as default prefix for all files in that folder, unless otherwise specified in an `index.js`. See "module.exports.autoPrefix" on how to overwrite this behaviour.
+
+```js
+// index.js
+fastify.register(AutoLoad, {
+  dir: path.join(__dirname, 'services'),
+  options: {}
+})
```

Comment:
Can we add an option to disable auto route prefixing? Or rather keep it off by default and enable it here with a flag say `autoRoutePrefix: true`?

Reply:
It's automatically disabled when an `index.js` is present in the folder..

Reply:
Not sure if that's explicit enough.

Reply:
I think it is quite explicit. The code is only executed:

- if there is a subfolder
- if the folder contains .js files
- if the folder does not contain an index.js file with further configuration
- if the file registers a route (fastify-plugin doesn't use prefix)

I've tested a bunch of existing use cases in my project and the current behaviour doesn't change. Auto-prefixing routes makes total sense to me, as it reduces the amount of additional configuration. If absolutely needed, I would enable it be default and add `autoRoutePrefix: false` to the options when to turn the feature off.

What do you think?